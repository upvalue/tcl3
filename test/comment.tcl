# hi this is a comment
