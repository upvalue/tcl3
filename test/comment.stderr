{"type": "TK_EOL", "begin": 0, "end": 0, "body": ""}
{"type": "TK_EOF", "begin": 0, "end": 0, "body": ""}
